js-escapes/eff.js

```diff
@@ -74,7 +75,8 @@
 				'quotes': 'single',
```

Comment:
This should be `'quotes': checkboxOutputJSON.checked ? 'double' : 'single'` but I’ll do that myself.